dev/src/index.ts

```diff
@@ -305,6 +305,23 @@ const MAX_CONCURRENT_REQUESTS_PER_CLIENT = 100;
  * @typedef {Object} BulkWriterOptions
  */
 
+/**
+ * An error thrown when a BulkWriter operation fails.
+ *
+ * The error used by {@link BulkWriter~shouldRetryCallback} set in
+ * {@link BulkWriter#onWriteError}.
+ *
+ * @property {GrpcStatus} code The status code of the error.
+ * @property {string} message The error message of the error.
+ * @property {DocumentReference} documentRef The document reference the operation was
+ * performed on.
+ * @property {'create' | 'set' | 'update' | 'delete'} operationType The type
```

Comment:
I assume you verified that this looks good in the generated docs.

Reply:
yep!